src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -838,10 +851,13 @@ void Graph::Allocate() {
 
 void Graph::CreatePrimitives() {
     OV_ITT_SCOPED_TASK(itt::domains::intel_cpu, "Graph::CreatePrimitives");
+    CREATE_DEBUG_TIMER(timer);
+
     for (auto& node : graphNodes) {
         OV_ITT_SCOPE(FIRST_INFERENCE, itt::domains::intel_cpu_LT, node->profiling.createPrimitive);
         DEBUG_LOG(*node);
         node->createPrimitive();
+        DEBUG_LOG_EXT("LOADTIME_createPrimitive", node->getName(), " ", node->getPrimitiveDescriptorType(), " ", timer.delta());
```

Comment:
Sorry I may need more time to refactor that since I'm not very good at writing complex macros, so I plan to just remove these new debug logs for this PR and leave it as a TODO item for me, Thx!